UPGRADE-4.3.md

```diff
@@ -160,6 +160,11 @@ SecurityBundle
  * Configuring encoders using `argon2i` as algorithm while only `argon2id` is
    supported is deprecated, use `argon2id` instead
 
+Serializer
+----------
+
+ * `AbstractNormalizer::createChildContext` now wants the format as 3rd parameter.
```

Comment:
let's remove this entry: no need to document changes to `@internal`

Reply:
removed